vignettes/datatable-importing.Rmd

```diff
@@ -82,6 +82,8 @@ test_that("generate dt", { expect_true(nrow(gen()) == 100) })
 test_that("aggregate dt", { expect_true(nrow(aggr(gen())) < 100) })
 ```
 
+If `data.table` is in Suggests (but not Imports) then you need to declare `.datatable.aware=TRUE` in one of the R/* files to avoid "object not found" errors when testing via `testthat::test_package` or `testthat::test_check`.
```

Comment:
could you confirm this statement is also valid when NAMESPACE is present? you can import anything from base R, or export dummy function there

Reply:
works with blank NAMESPACE, and when I do `importFrom("stats", "sd")`